### PR TITLE
:book: Update readme on test triggers on release-0.8

### PR DESCRIPTION
**Pull request by mquhuy**
This PR:

    Update build status badges
    Update test triggers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ development. Here are the tests which run on different branches in BMO:
 **E2E tests:**
 
 - **/test metal3-bmo-e2e-test-pull** runs required BMO e2e tests on your PR.
-  Works for main and release-0.5 branches.
+  Works for main, release-0.5, release-0.6 and release-0.8 branches.
```
Comment (tuminoid) [inline README.md:46]: We can drop this line, as we don't have 0.4 anymore where it didn't work.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Review by kashifest — COMMENTED** [review]
/approve

**Pull request merged**